README.md

```diff
@@ -1,75 +1,46 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Todo List Site
 
+> This site enables one have track of the activities to be done daily.
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+ - html
+ - javascript
+ - Webpack
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
+ In order to use the files, you will need to clone it and then follow the instructions below: 
+ 
+## Prerequisites
+- Good Knowlege of html.
+- Good knowledge of javascript.
+- Familiar with webpack.
+- Install nodejs then copy linters files from https://github.com/microverseinc/linters-config/tree/master/html-css-js
 
 ### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+Use 'git clone https://github.com/GordonOtieno/todo-list' to clone the repository to your local environment and begin working on it.
+- run **'npm init'** to install dependancy packages.
+- run **'npm run build'** to build the project and to generate dist files.
+- run **'npm run dev'** to run you project in development mode.
+ 
 
 ## Authors
 
-👤 **Author1**
+👤 **Gordon Otieno**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@GordonOtieno](https://github.com/GordonOtieno
+- Twitter: [@GordonO34459259](https://twitter.com/@GordonO34459259
 
-👤 **Author2**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+# screenshot.
+![Screenshot](show.jpg)
 
 ## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/GordonOtieno/concert/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
-
```

Comment:
- Please, kindly add the "Acknowledgment" section and its content
- Also, add the license and its contents